volasim_node/src/lee_control_ros1.cpp

```
@@ -71,55 +57,37 @@ void LeeControlNode::odom_cb(const nav_msgs::Odometry::ConstPtr& msg) {
-  desired_s.jerk = Eigen::Vector3d(pt.effort[Axis::X], pt.effort[Axis::Y],
-                                   pt.effort[Axis::Z]);
-
-  Eigen::Vector4d cmd = controller_.computeControls(state_, desired_s);
-
-  std_msgs::Float32MultiArray   msg;
-  std_msgs::MultiArrayDimension dim;
-  dim.size = Motors::N_MOTORS;
-  msg.layout.dim.push_back(dim);
+  double          t         = (ros::Time::now() - start_).toSec();
+  const auto&     desired_s = active_traj_.at_time(t);
+  Eigen::Vector4d cmd       = controller_.computeControls(state_, desired_s);
```

Clang (14.0.6)

[error] variable 'cmd' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)